cmd/terramate/e2etests/run_test.go

```diff
@@ -471,13 +489,14 @@ func TestCLIRunOrder(t *testing.T) {
 			},
 		},
 		{
-			name: "child stack can have explicit after clause to parent",
+			name: "child stack CANNOT have explicit after clause to parent",
```

Comment:
```suggestion
			name: "child stack cannot have explicit after clause to parent",
```

or maybe:

```suggestion
			name: "child stack with explicit after clause to parent fails",
```

Using all upper case seems quite off from how most tests are described on the project.

Reply:
it's because it was allowed before... tried to make it explicit the change. But I will change.